src/lab.rs

```diff
@@ -168,6 +168,23 @@ fn test_timeout(baseline_outcome: &Option<ScenarioOutcome>, options: &Options) -
     } else if options.baseline == BaselineStrategy::Skip {
         warn!("An explicit timeout is recommended when using --baseline=skip; using 300 seconds by default");
         Duration::from_secs(300)
+    } else if let Some(multiplier) = options.test_timeout_multiplier {
+        let timeout = max(
```

Comment:
Can you please change this to avoid the copy-paste with the following block?

i.e. if there is no fixed timeout, then:
the multiplier is the specified value or by default 5.0

Reply:
Makes sense, thanks. I'll do.